docs/source/api/python/nn-pytorch.rst

```diff
@@ -118,6 +118,7 @@ Utility Modules
     ~dgl.nn.pytorch.graph_transformer.DegreeEncoder
     ~dgl.nn.pytorch.utils.LaplacianPosEnc
     ~dgl.nn.pytorch.graph_transformer.BiasedMultiheadAttention
+    ~dgl.nn.pytorch.graph_transformer.GraphTransformerLayer
```

Comment:
Is the indent here correct?

Reply:
`GraphTransformerLayer` is right according to dongyu's design doc, and `LaplacianPosEnc` will be adjusted by dongyu.